crates/astria-eyre/Cargo.toml

```diff
@@ -8,8 +8,12 @@ edition = "2021"
 [dependencies]
 eyre = "0.6"
 itoa = "1.0.10"
+anyhow = { version = "1", optional = true }
```

Comment:
It's recommended to provide the exact version that you used when writing this code, i.e. the result of `cargo add anyhow --optional` in this case.